pypdf/generic/_data_structures.py

```diff
@@ -1416,7 +1416,7 @@ class Destination(TreeObject):
     node: Optional[
         DictionaryObject
     ] = None  # node provide access to the original Object
-    childs: List[
+    _filtered_children: List[
         Any
     ] = []  # used in PdfWriter - TODO: should be children  # noqa: RUF012
```

Comment:
We can probably remove the TODO comment here.

Another thing: Do we really need this at class level or can be move this and the initialization to the `__init__` method? The *RUF012* violation we silence here is about mutable defaults, which we would eliminate by this and thus avoid potential bugs.